cirq/optimizers/eject_phased_paulis.py

```diff
@@ -115,6 +115,8 @@ def _absorb_z_into_w(moment_index: int,
                      state: _OptimizerState) -> None:
     """Absorbs a Z^t gate into a W(a) flip.
 
+    [Where W(a) is shorthand for PhasedX(phase_exponent=a).]
```

Comment:
Optional: There seem to be a lot of places where we call this gate W. We're bound to miss some of the explanation of the symbol at some point. An alternative would be to adopt a short yet self-explanatory name such as "PhX" in place of W and remove the [Where...] comments. Your call.